teuthology/task/ceph_ansible.py

```diff
@@ -21,11 +21,21 @@ class CephAnsible(Task):
 
     _default_playbook = [
         dict(
+            hosts='all',
+            become=True,
+            tasks=[],
+        ),
```

Comment:
I think a simple comment here explaining the need for ansible to connect to all nodes at once and gather facts would be helpful.